profiles/clients.py

```diff
@@ -14,10 +14,9 @@ def __repr__(self) -> str:
 
 
 class Clients(collections.Set):
-    elements = []
```

Comment:
Seems that the list is static, and so every instance appends to it?

Reply:
that is a class-level variable, yes